### PR TITLE
Implemented Manage Keys and Download Postman Collection features for ETL/TASK Pipeline

### PR DESCRIPTION
**Pull request by tahierhussain**
## What

1. Implemented the Manage Keys feature.
2. Implemented the Download Postman Collection feature.
3. Added a new column to display the API endpoint in the table.

## Why

NA

## How

NA

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

This PR contains changes to the existing components, so there are chances of it breaking existing features. However, since the changes are not major, the possibility is low.

## Database Migrations

NA

## Env Config

NA

## Relevant Docs

NA

## Related Issues or PRs

https://github.com/Zipstack/unstract/pull/556

## Dependencies Versions

NA

## Notes on Testing

NA

## Screenshots
Added two new items to the actions dropdown: Manage Keys and Download Postman Collection.
Screenshot from 2024-08-07 00-41-16

Manage Keys Modal:
Screenshot from 2024-08-07 00-41-07

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: frontend/src/components/input-output/configure-ds/ConfigureDs.jsx [hunk 1]
```diff
@@ -82,6 +82,7 @@ function ConfigureDs({
   };
 
   const handleTestConnection = (updatedFormData) => {
+    console.log("HERE!!!");
```
Comment (Deepak-Kesavan) [inline frontend/src/components/input-output/configure-ds/ConfigureDs.jsx:85]: Remove this log.

File: frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx [hunk 2]
```diff
@@ -496,7 +586,7 @@ function Pipelines({ type }) {
           placement="bottomLeft"
           onOpenChange={() => setSelectedPorD(record)}
         >
-          <EllipsisOutlined rotate={90} className="p-or-d-actions" />
+          <EllipsisOutlined className="p-or-d-actions cur-pointer" />
```
Comment (Deepak-Kesavan) [inline frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx:589]: why is `rotate={90}` removed?

**Review by Deepak-Kesavan — CHANGES_REQUESTED** [review]

**Review by muhammad-ali-e — APPROVED** [review]
LGTM

File: frontend/src/components/input-output/configure-ds/ConfigureDs.jsx (hunk 1 above)
Comment (tahierhussain) [reply frontend/src/components/input-output/configure-ds/ConfigureDs.jsx]: @Deepak-Kesavan Thanks for the catch. Done.

File: frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx (hunk 2 above)
Comment (tahierhussain) [reply frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx:589]: @Deepak-Kesavan I removed it in order to maintain consistency with the API deployment page.

File: frontend/src/components/pipelines-or-deployments/pipeline-service.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { useAxiosPrivate } from "../../hooks/useAxiosPrivate";
+import { useSessionStore } from "../../store/session-store";
+
+function pipelineService() {
+  const axiosPrivate = useAxiosPrivate();
+  const { sessionDetails } = useSessionStore();
+  const path = `/api/v1/unstract/${sessionDetails.orgId.replaceAll('"', "")}`;
```
Comment (jagadeeswaran-zipstack) [inline frontend/src/components/pipelines-or-deployments/pipeline-service.js:7]: Let's have optional chaining for sessionDetails too. in order for it to be safe to have and consistent throughout the code.

File: frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx [hunk 4]
```diff
@@ -293,6 +316,49 @@ function Pipelines({ type }) {
...
+          onClick={() =>
+            downloadPostmanCollection(pipelineApiService, selectedPorD?.id)
+          }
+        >
+          <div>
+            <CloudDownloadOutlined />
+          </div>
+          <div>
+            <Typography.Text>Download Postman Collection</Typography.Text>
+          </div>
```
Comment (jagadeeswaran-zipstack) [inline frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx:356]: Do we need these div here? Can't we get the same result but adding styles to space component

File: frontend/src/components/pipelines-or-deployments/pipeline-service.js (hunk 3 above)
Comment (tahierhussain) [reply frontend/src/components/pipelines-or-deployments/pipeline-service.js]: @jagadeeswaran-zipstack Thanks for the catch. Done.

File: frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx (hunk 4 above)
Comment (tahierhussain) [reply frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx]: @jagadeeswaran-zipstack Good catch. I had refered this part of the code from `/deployments/api-deployment/ApiDeployment.jsx` component. So, I missed removing the wanted div tags. I have made the change now.

**Review by Deepak-Kesavan — APPROVED** [review]

**Review by jagadeeswaran-zipstack — APPROVED** [review]

**Pull request merged**